src/soep_preparation/clean_variables/pl.py

```diff
@@ -111,17 +111,15 @@ def clean(raw_data: pd.DataFrame) -> pd.DataFrame:
         renaming={"[2] Nein": False, "[1] Ja": True},
         ordered=True,
     )
-    out["mutterschaftsgeld_bezug_aktuell"] = object_to_bool_categorical(
+    out["bezieht_aktuell_mutterschaftsgeld"] = object_to_bool_categorical(
         series=raw_data["plc0152_v1"],
         renaming={"[1] Ja": True},
     )
-    out["mutterschaftsgeld_brutto_betrag_m_aktuell"] = object_to_float(
-        raw_data["plc0153_h"]
-    )
-    out["mutterschaftsgeld_betrag_durchschnittlich_m"] = object_to_float(
+    out["aktuelles_mutterschaftsgeld_m"] = object_to_float(raw_data["plc0153_h"])
+    out["durchschnittliches_mutterschaftsgeld_m"] = object_to_float(
         raw_data["plc0155_h"]
     )
```

Comment:
Used `bezogener_betrag` in this case to signal that it has been received (and **not** paid).
```suggestion
    out["bezogener_betrag_mutterschaftsgeld_letzer_monat"] = object_to_float(raw_data["plc0153_h"])
    out["bezogener_betrag_mutterschaftsgeld_m"] = object_to_float(
        raw_data["plc0155_h"]
    )
```

Reply:
You mean, you just removed "erhalten" from the variable names, correct? Here, we need a signal word about the direction of monetary flows, right?